src/com/vividsolutions/jump/workbench/ui/WorkbenchFrame.java

```diff
@@ -614,7 +613,7 @@ public void setMaximumFeatureExtentForEnvelopeRenderingInPixels(
      */
     @Deprecated
     public void log(String message) {
-        log(message, null, new Exception().getStackTrace()[0]);
+        Logger.info(message);
```

Comment:
please keep this, it points out from where the log() method call came

you can point log(String) to log(String,Class) though